File: assignment 16/p3.c
#include<stdio.h>
#include<stdlib.h>
void Display(int Arr[],int isize)
{
int icnt=0;
for(icnt=0;icnt<isize;icnt++)
{
if(((Arr[icnt]%5==0)&&(Arr[icnt]%2==0)))
{
printf("%d ",Arr[icnt]);
}
}
}
int main()
{
int icnt=0,ilength=0;
int *ptr=NULL;
printf("how many elements you want");
scanf("%d",&ilength);
ptr=(int*)malloc(sizeof(int)*ilength);
if(ptr==NULL)
{
	printf("unable malloc");
	return -1;
}
printf("enter elements");
for(icnt=0;icnt<ilength;icnt++)
{
scanf("%d",&ptr[icnt]);
}
Display(ptr,ilength);
free(ptr);
return 0;
}

File: assignment 32/P3.java
import java.util.*;
class Mangesh
{
	public void Display(int arr[],int ino)
	{
		int i=0,isize=0;
		isize=arr.length;
		for(i=0;i<isize;i++)
		{
			if(arr[i]%ino==0&&arr[i]%2==0)
			{
				System.out.println(arr[i]);
			}
			
		}
		
	}
	
}
class P3
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		int ilen=0, icnt=0,ino=0;
		System.out.println("enter divisor 1");
		ino=sc.nextInt();
		System.out.println("how many elements you want");
		ilen=sc.nextInt();
		int brr[]=new int[ilen];
		System.out.println("enter no");
		for(icnt=0;icnt<ilen;icnt++)
		{
			brr[icnt]=sc.nextInt();
		}
		Mangesh mobj= new Mangesh();
	   System.out.println("no divisable by divisor which are even are");
		mobj.Display(brr,ino);
		
	}
}